### PR TITLE
feat(xlm): CWC-183 support bip44 address generation

### PR DESCRIPTION
**Pull request by iancoolbitx**
xlm: change bip44 prefix 32 for 42, update get address and sign transaction
core: update get path

### CODE REVIEW

**Review by skchang0710 — DISMISSED** [review]

File: packages/core/src/coin/EDDSA.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as derivation from './derive';
 import Transport from "../transport";
 import * as utils from "../utils/index";
+import { pathType } from '../config/param'
```
Comment (delbinwen) [inline packages/core/src/coin/EDDSA.ts:4]: please add semicolon to keep coding style consistency...

File: packages/core/src/coin/EDDSA.ts [hunk 2]
```diff
@@ -21,7 +22,8 @@ export default class EDDSACoin {
    */
   async getPublicKey(transport: Transport, appPrivateKey: string, appId: string, isSLIP0010: boolean = true) {
     
-    const path = await utils.getPath(this.coinType, 0, 3, !isSLIP0010)
+    const pathprefix = isSLIP0010 ? pathType.SLIP0010 : pathType.BIP32_ED25519
```
Comment (delbinwen) [inline packages/core/src/coin/EDDSA.ts]:
```suggestion
    const pathPrefix = isSLIP0010 ? pathType.SLIP0010 : pathType.BIP32_ED25519
```

Comment (delbinwen) [reply packages/core/src/coin/EDDSA.ts]: camelCase variable naming is commonly adopted in JavaScript...

**Review by delbinwen — APPROVED** [review]
ok

**Review by skchang0710 — APPROVED** [review]

**Pull request merged**